compiler/rustc_resolve/src/build_reduced_graph.rs

```diff
@@ -322,9 +322,9 @@ impl<'a, 'ra, 'tcx> BuildReducedGraphVisitor<'a, 'ra, 'tcx> {
                 })
             }
             ast::VisibilityKind::Restricted { ref path, id, .. } => {
-                // For visibilities we are not ready to provide correct implementation of "uniform
+                // For restrictions we are not ready to provide correct implementation of "uniform
                 // paths" right now, so on 2018 edition we only allow module-relative paths for now.
-                // On 2015 edition visibilities are resolved as crate-relative by default,
+                // On 2015 edition restrictions are resolved as crate-relative by default,
```

Comment:
We are not yet changing visibilities. Let's revert those for now.

Reply:
Whoops, I didn't actually mean to change this :sweat_smile: I must've clicked the wrong place in the editor.